src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 From my last log above:
```powershell
[hevc_cuvid @ 00000000004d1a80] Bit depth 10 is not supported.
```

Every attempt I've made on the 10-bit file with harware decoding has resulted in the same error. The 8-bit file doesn't work with hardware decoding either though, it has a bunch of other errors shown above.